jobs/pacman-rule-engine-2.0/src/main/java/com/tmobile/pacman/util/AuditUtils.java

```diff
@@ -66,6 +68,7 @@ public static void postAuditTrail(List<Annotation> annotations, String status) {
                         CommonUtils.doHttpPost(requestUrl, requestBody.toString());
                     } catch (Exception e) {
                         logger.error("Audit creation failed", e);
+                        logger.error("Exception occurred in job with policyUUID:  Error message - Audit creation failed, "+e.getMessage());

```

Comment:
why we need this?

Reply:
gave reply above. removing the redundant logger message.